bindsnet/datasets/davis.py

```diff
@@ -57,6 +60,10 @@ def __init__(
         self.task = task
         self.subset = subset
         self.resolution = resolution
+        self.size = size
+
+        # Sets the boolean converted if the size of the images must be scaled down
+        self.converted = not (self.size == (600, 480))
```

Comment:
Nitpick: can be `self.converted = not self.size == (600, 480)`. Make sure you've installed the black pre-commit hook.

Reply:
I do have the hook installed, strange if it's suppose to catch this